Library/Formula/cucumber-cpp.rb

```diff
@@ -0,0 +1,24 @@
+class CucumberCpp < Formula
+  desc "Support for writing Cucumber step definitions in C++"
+  homepage "https://cucumber.io"
+  url "https://github.com/cucumber/cucumber-cpp/archive/1e633c6b59c75b959b480dcc4e40123e7fc59032.tar.gz"
+  version "0.3"
+  sha256 "1e211db8b8c230f291c8774d1c2e2bbbcf264d92016fd6081557425e43569c19"
+
+  depends_on "cmake" => :build
+  depends_on "ruby"
+
+  def install
+    system "gem", "install", "bundler"
+    system "cmake", "-DCUKE_DISABLE_GTEST=on",
+                    "-DCUKE_DISABLE_CPPSPEC=on",
+                    "."
+    system "cmake", "--build", "."
+    include.install "include/cucumber-cpp"
+    lib.install Dir["src/*.a"]
+  end
+
+  test do
+    system "false"
```

Comment:
@mikemcquaid It seems willing to do it on its own just fine. See below.
@MartinDelille Right. It handles that:

```
==> gem install bundler
Successfully installed bundler-1.11.2
Parsing documentation for bundler-1.11.2
Installing ri documentation for bundler-1.11.2
1 gem installed
==> /private/tmp/cucumber-cpp20160222-18652-ddfdea/cucumber-cpp-1e633c6b59c75b959b480dcc4e40123e7fc59032/.brew_home/bin/bundle
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies...
Rubygems 2.0.14 is not threadsafe, so your gems will be installed one at a time. Upgrade to Rubygems 2.1.0 or higher to enable parallel gem installation.
Installing ffi 1.9.10 with native extensions
Installing builder 3.2.2
Installing diff-lcs 1.2.5
Installing multi_json 1.11.2
Installing multi_test 0.1.2
Installing rspec-core 2.14.8
Installing rspec-mocks 2.14.6
Using bundler 1.11.2
Installing childprocess 0.5.9
Installing rspec-expectations 2.14.5
Installing gherkin 2.12.2 with native extensions
Installing rspec 2.14.1
Installing cucumber 1.3.20
Installing aruba 0.6.1
Bundle complete! 3 Gemfile dependencies, 14 gems now installed.
Bundled gems are installed into ./.brew_home.
==> /private/tmp/cucumber-cpp20160222-18652-ddfdea/cucumber-cpp-1e633c6b59c75b959b480dcc4e40123e7fc59032/.brew_home/bin/bundle show
Gems included by the bundle:
  * aruba (0.6.1)
  * builder (3.2.2)
  * bundler (1.11.2)
  * childprocess (0.5.9)
  * cucumber (1.3.20)
  * diff-lcs (1.2.5)
  * ffi (1.9.10)
  * gherkin (2.12.2)
  * multi_json (1.11.2)
  * multi_test (0.1.2)
  * rspec (2.14.1)
  * rspec-core (2.14.8)
  * rspec-expectations (2.14.5)
  * rspec-mocks (2.14.6)
```